packages/os/host-ctr-toml

```diff
@@ -1,6 +1,6 @@
 {{#if settings.container-registry.mirrors}}
```

Comment:
Would you mind showing this file in the testing too?

Reply:
Updated PR description with how `host-ctr.toml` renders for host-containers services.